carl_interactive_manipulation/package.xml

```diff
@@ -23,6 +25,9 @@
   <build_depend>rail_segmentation</build_depend>
   <build_depend>sensor_msgs</build_depend>
   <build_depend>wpi_jaco_msgs</build_depend>
+  <build_depend>std_msgs</build_depend>
+  <build_depend>visualization_msgs</build_depend>
+  <build_depend>actionlib_msgs</build_depend>
```

Comment:
make sure this matches the deps in the Cmake file. Looks like `urdf` is missing from quick glance.